hoot-core/src/main/cpp/hoot/core/io/OsmApiDbReader.cpp

```diff
@@ -158,18 +159,18 @@ void OsmApiDbReader::_readBounded(shared_ptr<OsmMap> map, const ElementType& ele
     // NODES
     ///////////////////////////////////////////////////////////////////
     case ElementType::Node:
-      while( elementResultsIterator->next() )
+      while (elementResultsIterator->next())
```

Comment:
maybe.  there wasn't too much of those, but should have done them outside of this pull request anyway.